static/sidebar/js/libs/player.js

```diff
@@ -44,13 +44,12 @@
         this.overlay = isset(options.overlay) ? options.overlay : null;
         this.overlayColor = isset(options.overlay_color) ? options.overlay_color : null;
         this.overlayOpacity = isset(options.overlay_opacity) ? options.overlay_opacity : null;
-        this.playButtonBackgroundColor = isset(options.play_button_background_color) ?  options.play_button_background_color : null;
-        this.playButtonBorderRadius = isset(options.play_button_border_radius) ? options.play_button_border_radius : null;
-        this.playButtonBorderWidth = isset(options.play_button_border_width) ? options.play_button_border_width : null;
-        this.playButtonBorderColor = isset(options.play_button_border_color) ? options.play_button_border_color : null;
-        this.playButtonIconColor = isset(options.play_button_icon_color) ? options.play_button_icon_color : null;
-        this.playButtonWidth = isset(options.play_button_width) ? options.play_button_width : null;
-        this.playButtonHeight = isset(options.play_button_height) ? options.play_button_height : null;
+        this.playButtonBackgroundColor = isset(options.item_play_button_background) ?  options.item_play_button_background : null;
+        this.playButtonBorderRadius = isset(options.item_play_button_border_radius) ? options.item_play_button_border_radius : null;
+        this.playButtonBorder = isset(options.item_play_button_border) ? options.item_play_button_border : null;
+        this.playButtonIconColor = isset(options.item_play_button_polygon_background) ? options.item_play_button_polygon_background : null;
+        this.playButtonWidth = isset(options.item_play_button_width) ? options.item_play_button_width : null;
+        this.playButtonHeight = isset(options.item_play_button_height) ? options.item_play_button_height : null;
```

Comment:
Please use the same names for options, only the formate will change.

good reference is https://github.com/tvpage/tvpage-hugo-template/blob/ac38b2771b6302830d9932ba75385aae450d77cc/static/solo-cta/js/libs/player.js

Example:

`this.playButtonIconColor = isset(options.play_button_icon_color) ? options.play_button_icon_color : null;`